lib/producer.js

```diff
@@ -137,7 +139,7 @@ function createRpcQueue(queue) {
   if (rpcQueues[queue].queue && !reconnection) return Promise.resolve();
 
   var resQueue = queue + ':' + process.env.HOSTNAME + ':res';
```

Comment:
Use a new uuid if HOSTNAME is not defined, to ensure people not using env var don't get collision